pulsar-client-tools-test/src/test/java/org/apache/pulsar/admin/cli/PulsarAdminToolTest.java

```diff
@@ -764,10 +764,7 @@ public void topics() throws Exception {
         cmdTopics.run(split("peek-messages persistent://myprop/clust/ns1/ds1 -s sub1 -n 3"));
         verify(mockTopics).peekMessages("persistent://myprop/clust/ns1/ds1", "sub1", 3);
 
-        cmdTopics.run(split("enable-deduplication persistent://myprop/clust/ns1/ds1"));
-        verify(mockTopics).enableDeduplication("persistent://myprop/clust/ns1/ds1", true);
-
-        cmdTopics.run(split("disable-deduplication persistent://myprop/clust/ns1/ds1"));
+        cmdTopics.run(split("set-deduplication persistent://myprop/clust/ns1/ds1 --disable"));
         verify(mockTopics).enableDeduplication("persistent://myprop/clust/ns1/ds1", false);
```

Comment:
since we refine "enable-deduplication" as "set-deduplication", we still have "get-deduplication-enabled" below. What's the differences?

Reply:
There is no corresponding query interface at the namespace-level, which needs to be added later